doc/whats_new/v1.6.rst

```diff
@@ -174,6 +174,9 @@ Changelog
 - |Fix| :class:`impute.KNNImputer` excludes samples with nan distances when
   computing the mean value for uniform weights.
   :pr:`29135` by :user:`Xuefeng Xu <xuefeng-xu>`.
+- |Fix| :class:`impute.IterativeImputer` no longer raises an error when `min_value` and `max_value`
+  are array-like and some features are dropped due to `keep_empty_features = False`.
+  :pr:`29451` by :user:`Guntitat Sawadwuthikul <gunsodo>`.
```

Comment:
You can also add here, that you fixed a bug when `max_value` and `min_value` would not index correctly when they are arrays and full nan columns are present.

Reply:
Added in 9a42943, thanks!